torcs doesn't use automake, so it shouldn't be calling the macro

http://bugs.gentoo.org/408473

--- torcs-1.3.6.orig/configure.in
+++ torcs-1.3.6/configure.in
@@ -15,9 +15,12 @@
 #
 ##############################################################################
 
-AC_INIT(Make-config.in)
+AC_INIT(torcs, 1.3.6)
 AC_CONFIG_HEADERS(config.h)
-AM_INIT_AUTOMAKE(torcs, 1.3.6)
+dnl Workaround code using old VERSION instead of new PACKAGE_VERSION
+VERSION=$PACKAGE_VERSION
+AC_DEFINE_UNQUOTED([VERSION], "$VERSION", [old VERSION workaround])
+AC_SUBST(VERSION)
 
 dnl Checks for programs.
 AC_PROG_CC
